packages/tools/client-debugger/client-debugger-view/webpack.config.js

```diff
@@ -46,8 +36,8 @@ module.exports = (env) => {
 				libraryTarget: "umd",
 			},
 			plugins: [
-				new webpack.ProvidePlugin({
-					process: require.resolve("process/browser"),
+				new HtmlWebpackPlugin({
```

Comment:
It's been a while since I've used this plugin, it looks like you're using it in place of polyfills from webpack right?